### ISSUE TITLE
 allow user to completely skip calibration

### ISSUE THREAD

**Issue by jidanni** [ShowMeHills/ShowMeHills#27]
These days often users purchase very good devices that don't need any calibration at all. Once presented with the confusing calibration window upon first starting, the user thanks can I please just get to the meat of the program? Also many users might be in a flat area just testing out the program, or at night time inside their house with no mountains to calibrate against. Anyway the default should just be that calibration is available on the menus if needed, perhaps. Or at least have the cardinal points, all four of them, available during calibration. At least I know where north south east and west are inside my house.

**Comment by jidanni** [comment]
Just be like
https://github.com/sky-map-team/stardroid
Which comes completely calibrated out of the box!

**Comment by jidanni** [comment]
By the way,, there is no way to get out of calibration. The menu bars are not there when you need them. No way to escape out. No way to find some help menu out of calibration.

**Comment by nikcain** [comment]
The calibration isn't for the compass direction, it's for the software to know the extent of the screen in the left-right direction. The size of the camera image varies from phone to phone, and it doesn't always match what's actually seen on the screen (sometimes it's clipped a bit). So the calibration screen allows the software to relate what the user's seeing to the direction they're facing in a way that represents what's actually seen on the screen.

This is only necessary because it's trying to place hill markers in as accurate a manner as possible. Apps like starmap aren't overlaying graphics exactly over a camera view, it just has to be in the right direction but not scaled to any particular field of view. So no need to have a fussy calibration.

I'll see what can be done about not being able to get out of calibration. I think first time the app is run it'll have to complete calibration, but if it's been done at least once then I agree it's reasonable to be able to cancel out of it.

**Comment by jidanni** [comment]
Yes, please have a big "just use the default" or "make a guess for me". And add a big " reset to default" rescue button. For example I have now done something wrong and everything is dancing all over the screen. I will have to wait for daytime and reinstall the entire app. Fortunately there are some mountains nearby which I can calibrate against. when I first encounter the calibrate screen I just tried to tap on it to get out of it but instead I've now screwed up the whole thing. PS my English isn't that bad, however I'm using Google Talk to Type thing.

**Comment by jidanni** [comment]
Okay I found what the problem is. The Fov calibration instructions on the screen should say that they are talking about common objects in your house like a bookcase or doorway, and not mountains listed on the screen! So there is no need for any Cardinal points that I mentioned previously. They should also mention that they are for use in doors were people are able to read the screen and its yellow arrows.

**Comment by jidanni** [comment]
Also be sure each item, including Compass, has a way to individually reset to original Factory or default state.
